lms/djangoapps/certificates/views/webview.py

```diff
@@ -420,25 +419,24 @@ def _render_certificate_template(request, context, course, user_certificate):
     return render_to_response("certificates/valid.html", context)
 
 
-def _update_microsite_context(context, configuration):
+def _update_configuration_context(context, configuration):
     """
-    Updates context with microsites data.
-    Microsites will need to be able to override any hard coded
+    Site Configuration will need to be able to override any hard coded
     content that was put into the context in the
     _update_certificate_context() call above. For example the
     'company_about_description' talks about edX, which we most likely
-    do not want to keep in a microsite
+    do not want to keep in configurations.
     So we need to re-apply any configuration/content that
     we are sourcing from the database. This is somewhat duplicative of
     the code at the beginning of this method, but we
     need the configuration at the top as some error code paths
     require that to be set up early on in the pipeline
     """
 
-    microsite_config_key = microsite.get_value('domain_prefix')
-    microsites_config = configuration.get("microsites", {})
-    if microsite_config_key and microsites_config:
-        context.update(microsites_config.get(microsite_config_key, {}))
+    config_key = configuration_helpers.get_value('domain_prefix')
+    config = configuration.get("microsites", {})
```

Comment:
@mattdrayer this is coming from database `CertificateHtmlViewConfiguration`, updating it would break existing functionality unless database entries are also updated.